mautic/mautic
# Removed the icon from drop list and from added filters rows.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ✅ ]
| New feature/enhancement? (use the a.x branch)      | [ ❌ ]
| Deprecations?                          | [ ❌ ]
| BC breaks? (use the c.x branch)        | [ ❌ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Removal of the icon for the filter field and list if they do not belong to lead and/or company object.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull it down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to segment, and click on New
3. Go to the filters tab
4. Search for Contact Behavior and Actions. The section is not with the icon
5. Now select one from the list, e.g., Asset Download; the added filter field should be without an icon.
